example/components/DepositETH.tsx

```diff
@@ -19,10 +25,13 @@ function DepositETHModal({ isOpen, onClose }: DepositETHModalProps) {
   const [amount, setAmount] = useState('')
   const [action, setAction] = useState<Action>('simulate')
   const { status: simulateStatus, refetch: simulateDepositETH } = useSimulateDepositETH({
+    // I agree that I prefer this to be flatter than this with no args but I also think there is inherent value
+    // in having fidelity with op-viem so I could go either way here
     args: {
       to: to as Address,
       gasLimit: 100000,
     },
+    // I question if users know what portal means. Might be good to call it portalAddress?
```

Comment:
Oh I remember now it's this way in op-viem because we are spreading addresses in {...opChainContracts} and portal is a property on that object